### PR TITLE
Fix propagation of custom attributes on outputs

### PR DESCRIPTION
**Pull request by AlanGriffiths**
Fixes: https://github.com/canonical/ubuntu-frame/issues/172

### CODE REVIEW

File: src/miral/static_display_config.cpp [hunk 1]
```diff
@@ -394,6 +394,11 @@ void miral::YamlFileDisplayConfig::serialize_output_configuration(
                 << "\n        group: " << conf_output.logical_group_id.as_value()
                 << "\t# Outputs with the same non-zero value are treated as a single display";
         }
+
+        for (auto const& [key, value] : conf_output.custom_attribute)
+        {
+            out << "\n        " << key << ": " << value.value_or("(nullopt)");
```
Comment (Saviq) [inline src/miral/static_display_config.cpp:400]: Should we even print this for unset keys?

Any of ``, `null`, `~` are YAML nulls, but I'm of two minds whether it should be printed at all if unset?

File: src/server/graphics/multiplexing_display.cpp [hunk 2]
```diff
@@ -91,7 +91,7 @@ class CompositeDisplayConfiguration : public mg::DisplayConfiguration
                     // MirSubpixelArrangement is mistakenly non-const?
                     output.gamma = mutable_output.gamma;
                     output.custom_logical_size = mutable_output.custom_logical_size;
-                    output.custom_attribute = std::move(mutable_output.custom_attribute);
+                    output.custom_attribute = mutable_output.custom_attribute;
```
Comment (RAOF) [inline src/server/graphics/multiplexing_display.cpp:94]: Ooof, this is a tricky one.

**Review by RAOF — APPROVED** [review]
LGTM.

**Review by Saviq — APPROVED** [review]
Noice!

**Pull request merged**